src/math/Interpolant.js

```diff
@@ -47,7 +47,7 @@ Interpolant.prototype = {
 
 		validate_interval: {
 
-			seek: {
+			seek: while(true) {
```

Comment:
Wow. I wasn't even aware Javascript could do this label stuff! The patch seems a bit hacky though...

@tschw Maybe this can be rewritten using functions/return and a closure?